src/infinity-menu.js

```diff
@@ -266,7 +278,7 @@ export default class InfinityMenu extends React.Component {
 	 *  @description React render method for creating infinity menu
 	 */
 	render() {
-		const tree = dcopy(this.props.tree);
+		const tree = this.props.tree;
```

Comment:
So I am adding `isSearchDisplay` and `isSearchOpen` to the tree to get rid of deep copy because searching is the only part we actually deleted things from the tree which was why we originally need the deep copy